common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1155,6 +1155,13 @@
     ],
     "sqlState" : "42623"
   },
+  "DESCRIBE_JSON_NOT_EXTENDED" : {
+    "message" : [
+      "DESCRIBE TABLE ... AS JSON only supported when [EXTENDED|FORMATTED] is specified.",
+      "For example: DESCRIBE EXTENDED TABLE AS JSON is supported but DESCRIBE TABLE AS JSON is not."
```

Comment:
If we pass the table name as a parameter we can actually generate the exact statement to issue as a fix, instead of "For example".